Adds new Meetup Organizers Toolkit (#9)
ansible-community/meetup: resources/email_templates/follow-up_email_for_the_speaker.md

> Hello {name},

Reviewer: ```suggestion
Hello {recipient's name},
```

Author: It  would be {your_name} , thank you  for pointing it out.

Reviewer 2: I think `{your_name}` is for organizer's name, which is not the same as `{recipient's name}`

This is {name} again.

> Have you had a chance to look at the email I sent on {date}? I am attaching the last email here for your reference.

Reviewer: ```suggestion
I'm {your name} and am writing to follow up about the speaking opportunity as part of an Ansible community event.

I was previously in touch and have included the previous email for your reference.
```

Reviewer: Maybe we could create a playbook with variables like `${email_recipient}` and `${your_name}` that generates all the .txt file templates.

Author: That would be good greatly helpful :)

Please let me know if you need further information.

> We are looking forward to your reply.

Reviewer: ```suggestion
We look forward to hearing from you soon.
```

Author: Updated.

> Thank you

Reviewer: ```suggestion
Thank you,
```

Author: updated.
